doc/metric_spec.md

```diff
@@ -249,3 +249,42 @@ Either the value is `1`, or the line is absent altogether.
 - `disk_state`: one of `attaching|failed|negotiating|inconsistent|outdated|dunknown|consistent|uptodate`
 
 The total number of lines for this metric will be the cardinality of `name` times the cardinality of `volume`.
+
+### `ha_cluster_drbd_split_brain`
+
+#### Description
+
+This metric signal if there is a split brain occuring per resource and volume.
+Either the value is `1`, or the line is absent altogether.
+
+This metric is a special metric comparing to others, because in order to make this metric working you will need to set a drbd customer split-brain handler. Look at the end 
+
+### Labels
+
+- `resource`: the name of the resource.
+- `volume`: the volume number
+
+### Setting up the DRBD split brain hook:
+
+In order to get woring the split_brain metric:
+
+1) create hook:
+
+```bash
+#! /bin/bash
+mkdir -p /var/run/drbd/splitbrain
+echo "DRBD split-brain detected! Remember to remove the file /var/lib/drbd/drbd-split-brain-detected once the splitbrain is solved!" > /var/run/drbd/splitbrain/drbd-split-brain-detected-$DRBD_RESOURCE-$DRBD_VOLUME
+```
+
+2) copy this hook into all drbd nodes:
+```/var/lib/drbd/notify-split-brain-haclusterexporter-suse-metric.sh```
+
+3) on the drbd configuration enable the hook:
+
+```split_brain: "/usr/lib/drbd/notify-split-brain-haclusterexporter-suse-metric.sh"`
+
+Refer to upstream doc: https://docs.linbit.com/docs/users-guide-8.4/#s-configure-split-brain-behavior
```

Comment:
Maybe it's worth mentioning the full solution documentation (https://github.com/SUSE/ha-sap-terraform-deployments/blob/develop/doc/monitoring.md#drbd-split-brain-detection) so that users understand they don't need to do any of these steps if they're using the exporter within the full solution.

Reply:
ok